### ISSUE TITLE
Robovac Error

### ISSUE THREAD

**Issue by mirunamd** [apexad/homebridge-eufy-robovac#41]
I'm getting the following error in the logs:

 [homebridge-eufy-robovac] This plugin slows down Homebridge. The read handler for the characteristic 'On' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.

Connecting...
Robovac Error {}
Disconnected!
Connecting...
Robovac Error {}
Disconnected!

Is there anything I can do to diagnose this?

**Comment by bursaar** [comment]
Getting the same, and the Eufy vac has stopped responding.

**Comment by greyivy** [comment]
Same error. The plugin is completely broken for me.

**Comment by martsie** [comment]
Yep same.

**Comment by apexad** [comment]
Sorry all. My Robovac is in storage and even beyond that I regrettably don’t have much time these days.

**Comment by lawrencegripper** [comment]
This fork has the IP option and worked for me: https://github.com/ale770/homebridge-eufy-robovac

**Comment by tl-mueller** [comment]
I also tried using the repository of ale770:

[12/10/2022, 7:42:03 PM] [Vacuum Cleaner] Launched child bridge with PID 3313907
[12/10/2022, 7:42:03 PM] Registering accessory '@ale770/homebridge-eufy-robovac.Eufy RoboVac'
[12/10/2022, 7:42:03 PM] [Vacuum Cleaner] Loaded @ale770/homebridge-eufy-robovac v1.1.3 child bridge successfully
[12/10/2022, 7:42:03 PM] Loaded 0 cached accessories from cachedAccessories.0E4F85ABA2ED.
[12/10/2022, 7:42:03 PM] [Vacuum Cleaner] Vacuum Cleaner finished initializing!
[12/10/2022, 7:42:04 PM] Homebridge v1.6.0 (HAP v0.11.0) (Vacuum Cleaner) is running on port 55313.
[12/10/2022, 7:42:05 PM] Registering platform 'homebridge-eufy-security.EufySecurity'

When I try to turn the vacuum cleaner on, it ends in a no response error. Is there anything else that can be done? I added the Device ID, Local ID (using the Python grabber https://gitlab.com/Rjevski/eufy-device-id-and-local-key-grabber) and internal IP address. 

Does anyone has an idea how I can get Eufy RoboVac up and running in Homebridge to use it in the Home app?
